hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/AbstractS3ATestBase.java

```diff
@@ -159,4 +168,63 @@ protected void assertStatusCode(AWSServiceIOException e, int code)
       throw e;
     }
   }
+
```

Comment:
I think we should actually have this in a new class, say, EncryptionTestUtils; we can add more as and when needed. I'd have suggested S3ATestUtils, but that's too much source of cherrypick/merge pain these days, which is why I'm trying to split work up